packages/api/src/Base.ts

```diff
@@ -101,6 +101,7 @@ export default abstract class ApiBase<CodecResult, SubscriptionResult> implement
     this._rpcRx = new RpcRx(options.provider);
     this._rpc = this.decorateRpc(this._rpcRx, this.onCall);
     this._rx.rpc = this.decorateRpc(this._rpcRx, rxOnCall);
+    this._rx.signer = options.signer;
```

Comment:
imo signer has nothing to do with rx, so it shouldn't be in rx. I would propose `this._signer = options.signer` here

Reply:
it is more like `_rx` is not a proper name, because it's the api object that is passed to SubmittableExtrinsic.